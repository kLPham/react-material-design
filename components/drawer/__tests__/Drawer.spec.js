import React from 'react';
import { shallow } from 'enzyme';
import Drawer from '../index';

describe('<Drawer />', () => {
    it('should render', () => {
        const drawer = shallow(
            <Drawer />,
    );
        expect(drawer).toMatchSnapshot();
    });
});
