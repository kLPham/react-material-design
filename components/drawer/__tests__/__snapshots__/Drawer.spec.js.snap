// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Drawer /> should render 1`] = `
<aside
  className="mdc-permanent-drawer mdc-typography"
/>
`;
